bundles/TinymceBundle/public/assets/tinymce.js

```diff
@@ -16,4 +16,10 @@ import 'tinymce/plugins/insertdatetime';
 import 'tinymce/plugins/media';
 import 'tinymce/plugins/help';
 import 'tinymce/plugins/lists';
-import 'tinymce/plugins/code';
\ No newline at end of file
+import 'tinymce/plugins/code';
+
+import 'tinymce/plugins/searchreplace';
+import 'tinymce/plugins/advlist';
+import 'tinymce/plugins/charmap';
+import 'tinymce/plugins/hr';
```

Comment:
This plugin is missing
```suggestion
```

Reply:
@Daniel-Ateles Could you apply this change?

Reply:
> @Daniel-Ateles Could you apply this change?

Done!